.github/workflows/test.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Test'
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+
+jobs:
+  analyze:
+    uses: UKHomeOffice/hocs-ci-infrastructure/.github/workflows/test.yml@v1
+    with:
+      components: 'localstack postgres'
+      elastic: 'false'
```

Comment:
would be unsurprised if this evaluates to true with the quotemarks

Reply:
Yep I want this to work with environment variables instead, this should be a draft Pr really.